pytorch_toolkit/formula_recognition/README.md

```diff
@@ -0,0 +1,159 @@
+# PyTorch realization of the Im2Markup
+
+This repository contains inference and training code for Im2LaTeX models.
+Source [repository](https://github.com/harvardnlp/im2markup/). This repository is a fork of [PyTorch realization](https://github.com/luopeixiang/im2latex/)
+Models code is designed to enable ONNX\* export and inference on CPU via OpenVINO™.
+
+## Setup
+
+### Prerequisites
+
+* Ubuntu\* 16.04
+* Python\* 3.7 or newer
+* PyTorch\* (1.4.0)
+* CUDA\* 10.1
+* OpenVINO™ 2020.1 with Python API
```

Comment:
too old already

Reply:
What exactly is too old?

Reply:
OpenVINO version here is outdated. Please put a modern one (I hope it's compatible with the code).

Reply:
Done